scripts/build-windows-k8s.sh

```diff
@@ -346,8 +320,9 @@ copy_dockerfile() {
 }
 
 create_zip() {
+	ZIP_NAME="${k8s_e2e_upstream_version:-"v${ACS_VERSION}int.zip"}"
```

Comment:
@JiangtianLi Made a small update here to allow upstream versions to not be limited to vnumberint.zip naming convention.